app/home/index.jsx

```diff
@@ -57,6 +70,16 @@ class HomeContainer extends React.PureComponent {
     );
   }
 
+  handleScrollEvent() {
+    const { dispatch } = this.props;
+    const top = (document.documentElement && document.documentElement.scrollTop) || document.body.scrollTop;
```

Comment:
'document' object doesn't exist on NodeJS.
So, this code brake the whole ServerSideRendering. 

This code need EnvChecker.isServer() to avoid a crash